wiki/Contributor's-Guide.md

```diff
@@ -85,6 +86,23 @@ In the [Workflow](Workflow) article, there is a section detailing the [GitHub Wo
 
 Note: If changes are made that intentionally modify the generated documentation/code, it is imperative to [update the stable folder](https://github.com/JacquesCarette/Drasil/wiki/Workflow#updating-stable-folder) in order to prevent any CI errors.
 
+## Git Best Practices
+To maintain a clean, understandable project history and improve collaboration, please follow these guidelines when contributing code:
+- Structure commits thoughtfully
+  - Group related changes into the same commit making them easier to track. Ask yourself if the changes you are committing serve the same coherent purpose. If so, try to commit them all together.
```

Comment:
I would phrase this in the negative: Do not put unrelated items in a single PR. I actually prefer multiple small PRs.